pkgs/development/compilers/elm/packages/ghc9_2/elm-format/elm-format.nix

```diff
@@ -24,18 +24,8 @@ mkDerivation rec {
     quickcheck-io relude tasty tasty-hspec tasty-hunit tasty-quickcheck
     text
   ];
-  doHaddock = false;
   homepage = "https://elm-lang.org";
   description = "A source code formatter for Elm";
   license = lib.licenses.bsd3;
   mainProgram = "elm-format";
-  postPatch = ''
-    mkdir -p ./generated
-    cat <<EOHS > ./generated/Build_elm_format.hs
-    module Build_elm_format where
-
-    gitDescribe :: String
-    gitDescribe = "${version}"
-    EOHS
-  '';
```

Comment:
I think this is probably requored for elm-format to report correct version. Did you test this is really not needed?

cc @avh4

Reply:
We're still applying this, but in the override instead, to keep this file as clean as cargo2nix generated.